.github/workflows/linux_builds.yml

```
@@ -62,6 +62,11 @@ jobs:
         run: |
           scons -j2 verbose=yes warnings=all werror=yes platform=linuxbsd tools=yes target=release_debug module_mono_enabled=yes mono_glue=no
 
+      # Execute unit tests for the editor
+      - name: Unit Tests
+        run: |
+          ./bin/godot.linuxbsd.opt.tools.64 --test
```

This now needs to be:
```suggestion
          ./bin/godot.linuxbsd.opt.tools.64.mono --test
```

Only for Linux, Mono builds are not done on other platforms for now.